Assets/Dojo/Runtime/SynchronizationMaster.cs

```diff
@@ -133,5 +134,23 @@ public void RegisterEntityCallbacks()
         {
             ToriiEvents.Instance.OnEntityUpdated += HandleEntityUpdate;
         }
+
+        private ModelInstance[] LoadModels()
+        {
+            List<ModelInstance> models = new();
+
+            Assembly assembly = Assembly.GetAssembly(typeof(ModelInstance));
+            var modelTypes = assembly.GetTypes()
+                .Where(t => typeof(ModelInstance).IsAssignableFrom(t) && t.IsAbstract == false);
+
+            GameObject go = new("Models");
```

Comment:
Does this create a new Models game object that contains the models as components? I think it's fine too but ultimately we want the models game object to be a child of the world manager

Reply:
I tried to make the models game object a child of the world manager but an additional capsule was created so now I added the models as components of the world manager